cypress/support/users.js

```diff
@@ -0,0 +1,26 @@
+
+export const users = {
```

Comment:
Try to view a module like this one, as an actual object that can send/receive messages such that the properties or function names are not so redundant. For example: Think of a dog. You know a dog has a breed, name, it can bark, walk, etc. 
Then we can have a module as:
```javascript
export const dog = {
  name: 'Bob',
  breed: 'Beagle',

  bark: () => console.log('woof woof'),
  walk: () => console.log("I'm walking")
}

// When I call this object's properties and methods, it is not redundant and it's actually more readable
console.log(dog.name);
dog.bark();
```

Compare this to something like `dog.dogName` or `dog.dogBark()` which is more redundant